### DISCUSSION TITLE
Is there a way to scroll only a specific container (rather than the entire page)?

### DISCUSSION THREAD

**Discussion by mybearworld** [philc/vimium#4573] [General]
This is probably a really basic question, but I can't seem to find a way to do this. How would I scroll a specific overflowing container?

**Comment by philg-dev** [comment]
We've had a discussion on this topic just the other day in this issue:

https://github.com/philc/vimium/issues/4551

TL;DR: seems like it's currently not possible purely with the keyboard.

**Reply by mybearworld** [reply]
That's unfortunate. Thank you, though!
